File: platform-mobile/src/main/java/ru/pocketbyte/locolaser/platform/mobile/IosSwiftPlatformConfig.java
package ru.pocketbyte.locolaser.platform.mobile;

import ru.pocketbyte.locolaser.platform.mobile.resource.IosSwiftResources;
import ru.pocketbyte.locolaser.resource.PlatformResources;

public class IosSwiftPlatformConfig extends IosBaseClassPlatformConfig {

    public static final String TYPE = "ios_swift";

    @Override
    public String getType() {
        return TYPE;
    }

    @Override
    public PlatformResources getResources() {
        return new IosSwiftResources(getResourcesDir(), getResourceName(), getTableName());
    }

}

File: platform-mobile/src/main/java/ru/pocketbyte/locolaser/platform/mobile/IosSwiftPlatformConfig.kt
package ru.pocketbyte.locolaser.platform.mobile

import ru.pocketbyte.locolaser.platform.mobile.resource.IosSwiftResources
import ru.pocketbyte.locolaser.resource.PlatformResources

class IosSwiftPlatformConfig : IosBaseClassPlatformConfig() {

    companion object {
        const val TYPE = "ios_swift"
    }

    override val type = TYPE
    override val resources
        get() = IosSwiftResources(resourcesDir!!, resourceName!!, tableName, filter)

}
